routers/routes/routes.go

```diff
@@ -467,6 +467,19 @@ func RegisterRoutes(m *macaron.Macaron) {
 				m.Post("/content", repo.UpdateIssueContent)
 				m.Post("/watch", repo.IssueWatch)
 				m.Combo("/comments").Post(bindIgnErr(auth.CreateCommentForm{}), repo.NewComment)
+				m.Group("/times", func() {
+					m.Post("/add", repo.AddTimeManual)
+					m.Group("/stopwatch", func() {
+						m.Get("/toggle", repo.IssueStopwatch)
```

Comment:
Upps, I thought I was at the API router but it is at normal router. Then it is no problem fixing it.